frontend/src/turbo/aria-stream-action.ts

```diff
@@ -0,0 +1,19 @@
+import { StreamActions, StreamElement } from '@hotwired/turbo';
+import { announce } from '@primer/live-region-element';
+
+export function registerAriaStreamAction() {
+  StreamActions.aria = function ariaStreamAction(this:StreamElement) {
+      const message = this.getAttribute('message') ?? '';
+      const type = this.getAttribute('type') ?? 'polite';
```

Comment:
Logical OR here is probably better here - so that we also handle blank attribute values, e.g. `type=""`.

```suggestion
      const message = this.getAttribute('message') || '';
      const type = this.getAttribute('type') || 'polite';
```

Reply:
Perhaps we could also call this "politeness" rather than the much more generic "type"?